### PR TITLE
Mr mask

### PR DESCRIPTION
**Pull request by TwinkmrMask**
let's try it

### CODE REVIEW

File: csharp/Platform.Numbers/Arithmetic[T].cs [hunk 1]
```diff
@@ -4,13 +4,10 @@
 using Platform.Exceptions;
 using Platform.Reflection;
 
-// ReSharper disable StaticFieldInGenericType
```
Comment (Konard) [inline csharp/Platform.Numbers/Arithmetic[T].cs:7]: 7-th line should be preserved.

File: csharp/Platform.Numbers/Bit[T].cs [hunk 2]
```diff
@@ -9,50 +9,50 @@
 namespace Platform.Numbers
 {
     /// <summary>
-    /// <para>.</para>
-    /// <para>.</para>
+    /// <para>Represents a set of compiled bit operation delegates.</para>
```
Comment (Konard) [inline csharp/Platform.Numbers/Bit[T].cs:12]: `operations`

File: csharp/Platform.Numbers/Bit[T].cs [hunk 3]
```diff
@@ -9,50 +9,50 @@
...
+    /// <para>Представляет набор скомпилированных делегатов битовых операций.</para>
     /// </summary>
     public static class Bit<T>
     {
         /// <summary>
-        /// <para>.</para>
-        /// <para>.</para>
+        /// <para>A read-only field that represents a delegate of the function of the bitwise inversion of a number.</para>
```
Comment (Konard) [inline csharp/Platform.Numbers/Bit[T].cs:18]: `delegate of the function of the bitwise inversion` ⇾ `bitwise inversion function delegate`

**Review by Konard — COMMENTED** [review]
This way it is better.

File: csharp/Platform.Numbers/Bit[T].cs [hunk 4]
```diff
@@ -9,50 +9,50 @@
...
-        /// <para>.</para>
+        /// <para>A read-only field that represents a logic multiplication function delegate.</para>
+        /// <para>Поле только для чтения, представляющее делегат функции логического умножения.</para>
         /// </summary>
         public static readonly Func<T, T, T> And = CompileAndDelegate();
 
         /// <summary>
-        /// <para>.</para>
-        /// <para>.</para>
+        /// <para>A read-only field that represents a bitwise shifting a number to the left function delegate.</para>
```
Comment (Konard) [inline csharp/Platform.Numbers/Bit[T].cs:36]: `shifting a number to the left` ⇾ `left shift`

File: csharp/Platform.Numbers/Bit[T].cs [hunk 5]
```diff
@@ -9,50 +9,50 @@
...
+        /// <para>Поле только для чтения, представляющее делегат функции побитового сдвига числа влево.</para>
         /// </summary>
         public static readonly Func<T, int, T> ShiftLeft = CompileShiftLeftDelegate();
 
         /// <summary>
-        /// <para>.</para>
-        /// <para>.</para>
+        /// <para>A read-only field that represents a bitwise shifting a number to the right function delegate.</para>
```
Comment (Konard) [inline csharp/Platform.Numbers/Bit[T].cs:42]: `shifting a number to the right` ⇾ `right shift`

File: csharp/Platform.Numbers/Bit[T].cs [hunk 6]
```diff
@@ -9,50 +9,50 @@
...
+        /// <para>Поле только для чтения, представляющее делегат функции побитового сдвига числа вправо.</para>
         /// </summary>
         public static readonly Func<T, int, T> ShiftRight = CompileShiftRightDelegate();
 
         /// <summary>
-        /// <para>.</para>
-        /// <para>.</para>
+        /// <para>A read-only field that represents a partial number rewrite function delegate.</para>
+        /// <para>Поле только для чтения, представляющее делегат функции частичной перезаписи числа.</para>
```
Comment (Konard) [inline csharp/Platform.Numbers/Bit[T].cs:49]: `функции частичной побитовой записи`

**Review by Konard — COMMENTED** [review]
Final touches.

File: csharp/Platform.Numbers/Bit[T].cs (hunk 5 above)
Comment (TwinkmrMask) [reply csharp/Platform.Numbers/Bit[T].cs:42]: right shift number, maybe?

**Pull request merged**